doc/HalideCMakePackage.md

```diff
@@ -474,7 +474,9 @@ verbatim.
 If `GRADIENT_DESCENT` is set, then the module will be built suitably for
 gradient descent calculation in TensorFlow or PyTorch. See
 `Generator::build_gradient_module()` for more documentation. This corresponds to
-passing `-d 1` at the generator command line.
+passing `-d 1` at the generator command line. Note that you *must* also specify
+an autoscheduler (via the `AUTOSCHEDULER` argument) when specifying this
+argument.
```

Comment:
We should check this in HalideGeneratorHelpers.cmake

Reply:
First we should get buy-in from everyone as to whether this change is a good one.